environments/staging/aws-resources.yml

```diff
@@ -8,6 +8,11 @@ couch15-staging: 10.201.42.109
 couch15-staging.instance_id: i-0f618ee36887bb494
 couch6-staging: 10.201.40.74
 couch6-staging.instance_id: i-01b32f8a2eebe07bc
+couch7-staging: 10.201.40.221
+couch7-staging.instance_id: i-0361f071191cd66c0
+couch8-staging: 10.201.40.18
+couch8-staging.instance_id: i-06d9f01ac9c8cb6b8
```

Comment:
This looks like an unrelated change (I think these are the marches that you already terminated?)